gcc/rust/ast/rust-ast-dump.cc

```diff
@@ -1171,6 +1182,8 @@ Dump::visit (TypeAlias &type_alias)
     visit (type_alias.get_where_clause ());
   stream << " = ";
   visit (type_alias.get_type_aliased ());
+  stream << ";"
+	 << "\n";
```

Comment:
Same, here you can just merge the two strings
```suggestion
  stream << ";\n";
```